extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -273,6 +290,81 @@ export class AmpAnalytics extends AMP.BaseElement {
       return val + argList;
     });
 
+    if (request.startsWith("iframe")) {
+      this.handleIframe_(request, trigger, unusedEvent);
+    }
+    else {
+      this.handleGet_(request, trigger, unusedEvent);
+    }
+  }
+
+  /**
+   * Callback for events that are registered by the config's iframe triggers.
+   * This method generates the request and sends the request out.
+   *
+   * @param {!JSONObject} request string which contains the url to be loaded
+   *                              into the iframe
+   * @param {!JSONObject} trigger JSON config block that resulted in this event.
+   * @param {!Object} unusedEvent Object with details about the event.
+   * @private
+   */
+  handleIframe_(request, trigger, unusedEvent) {
```

Comment:
Yeah I struggled with that. My reasoning is that the unusedEvent gets passed around in private methods and isn't actually used in any of them, so in deference to the original intent I left it as is.